### ISSUE TITLE
Remove warnings for unlisted files

### ISSUE THREAD

**Issue by martinfiebig** [asam-ev/OpenMATERIAL#123]
**Describe the bug**
During the build of the documentation, Antora generates warnings for unlisted files (see below screenshot). This only means, that the files are not referenced in the nav.adoc file. These warnings are not critical, but should not appear in the log.

**Expected behavior**
Investigate possibilities to disable/configure these warnings.

**Screenshots**
<img width="704" alt="image" src="https://github.com/user-attachments/assets/18301b1d-411b-4a78-8e6e-34f0c6256823">

**Comment by martinfiebig** [comment]
Hi @ClemensLinnhoff,
according to the [documentation](https://code.asam.net/common/asam-antora-extensions#user-content-orphan-pages-and-partials) of the asam-antora-extensions, the extension for finding and listing the orphan-pages is always active and can not be disabled completely. However, you can set a parameter 'orphan_exceptions' to exclude certain paths from this extension. I've tested this in my forked repository, and set the root-folder as exception-path, which effectively disables the extension (see below screenshot, the step seems to be executed, but the warnings are gone). This way, we don't need to adapt and possibly maintain the files in the asam-antora-extensions submodule and also don't need to change the structure of our content.

<img width="360" alt="image" src="https://github.com/user-attachments/assets/36745028-16a0-4c59-8800-d0212c2ad3fc">

If you think this is an acceptable solution, I'll create a pull request for the necessary change.

**Comment by ClemensLinnhoff** [comment]
Yes, I think this is okay, since we only have the main sections in the nav bar anyways. And most likely in the future mainly the subsections will be changed and not the main ones. So this orphan detector is not of much use to us anyways.

**Comment by martinfiebig** [comment]
Agreed. I've created the pull request #125 to solve this issue.
